base_report_to_printer/models/ir_actions_report.py

```diff
@@ -159,14 +159,14 @@ def _render_qweb_pdf(self, report_ref, res_ids=None, data=None):
         document, doc_format = super()._render_qweb_pdf(
             report_ref=report_ref, res_ids=res_ids, data=data
```

Comment:
As report_ref is not a named argument, I would also remove "report_ref=" in the super()

Reply:
Done :+1: